package.json

```diff
@@ -6,12 +6,15 @@
   "scripts": {
     "start": "node index.js",
     "test": "cross-env NODE_ENV=test mocha ./test/**/*.js",
-    "coverage": "cross-env NODE_ENV=test ./node_modules/istanbul/lib/cli.js cover ./node_modules/mocha/bin/_mocha",
-    "lint": "eslint ."
+    "coverage": "npm run pretest && cross-env NODE_ENV=test ./node_modules/istanbul/lib/cli.js cover ./node_modules/mocha/bin/_mocha && npm run posttest",
+    "pretest": "sequelize --env test db:migrate && sequelize --env test db:seed",
```

Comment:
You should undo migration before running `db:migrate` to ensure the tables are dropped before recreating. I suggest you edit the `pretest` script to look like this `npm run posttest && sequelize --env test db:migrate && sequelize --env test db:seed`